endpoint-root/endpoint-jersey/src/main/java/org/divy/common/bo/endpoint/jersey/hypermedia/association/JerseyAssociationsHandler.java

```diff
@@ -0,0 +1,32 @@
+package org.divy.common.bo.endpoint.jersey.hypermedia.association;
+
+import org.divy.common.bo.BusinessObject;
+import org.divy.common.bo.endpoint.hypermedia.association.Association;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.divy.common.bo.endpoint.hypermedia.association.Association'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)